@xen-orchestra/lite/src/views/pool/PoolNetworkView.vue

```diff
@@ -1,25 +1,45 @@
 <template>
   <div class="pool-network-view">
     <UiCard class="container">
-      <PoolNetworksTable />
-      <PoolHostInternalNetworksTable />
+      <PoolNetworksTable :networks="networksWithPifs" />
+      <PoolHostInternalNetworksTable :networks="networksWithoutPifs" />
     </UiCard>
+    <PoolNetworksSidePanel v-if="network" :network />
+    <UiPanel v-else>
+      <VtsNoSelectionHero type="panel" />
+    </UiPanel>
   </div>
 </template>
 
 <script lang="ts" setup>
 import PoolHostInternalNetworksTable from '@/components/pool/network/PoolHostInternalNetworksTable.vue'
+import PoolNetworksSidePanel from '@/components/pool/network/PoolNetworksSidePanel.vue'
 import PoolNetworksTable from '@/components/pool/network/PoolNetworksTable.vue'
 import { usePageTitleStore } from '@/stores/page-title.store'
+import { useNetworkStore } from '@/stores/xen-api/network.store'
+import VtsNoSelectionHero from '@core/components/state-hero/VtsNoSelectionHero.vue'
 import UiCard from '@core/components/ui/card/UiCard.vue'
+import UiPanel from '@core/components/ui/panel/UiPanel.vue'
+import { useRouteQuery } from '@core/composables/route-query.composable'
+import { computed } from 'vue'
 import { useI18n } from 'vue-i18n'
 
 usePageTitleStore().setTitle(useI18n().t('network'))
+
+const { records: networks, networksWithPifs, networksWithoutPifs } = useNetworkStore().subscribe()
+
+const networkId = useRouteQuery('id')
+
+const network = computed(() => networks.value.find(network => network.uuid === networkId.value))
```

Comment:
You can use the `useRouteQuery` transformers:

```suggestion
const network = useRouteQuery<XenApiNetwork | undefined>('id', {
  toData: id => getByUuid(id as XenApiNetwork['uuid']),
  toQuery: network => network?.uuid ?? '',
})
```

For now `toQuery` is required, even if in this case it's useless. (I'll probably change that some day but it's not urgent)